Date: Sat, 30 Nov 2013 13:00:06 -0500
From: Kevin O'Connor <kevin@koconnor.net>
To: seabios@seabios.org
Message-ID: <05c9516da4b27910c697509d11a5e806d6fe89d4.1385834345.git.kevin@koconnor.net>
Subject: [SeaBIOS] [PATCH 3/5] vgabios: Add bochsvga "HDTV" resolutions.

The same set of video modes were added to the "lgpl vgabios" in
release 0.7a.

Signed-off-by: Kevin O'Connor <kevin@koconnor.net>
---
 vgasrc/bochsvga.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/vgasrc/bochsvga.c b/vgasrc/bochsvga.c
index 27f7615..a19eccb 100644
--- a/vgasrc/bochsvga.c
+++ b/vgasrc/bochsvga.c
@@ -70,2 +70,5 @@ static struct bochsvga_mode
     { 0x14c, { MM_DIRECT, 1152, 864,  32, 8, 16, SEG_GRAPH } },
+    { 0x175, { MM_DIRECT, 1280, 768,  16, 8, 16, SEG_GRAPH } },
+    { 0x176, { MM_DIRECT, 1280, 768,  24, 8, 16, SEG_GRAPH } },
+    { 0x177, { MM_DIRECT, 1280, 768,  32, 8, 16, SEG_GRAPH } },
     { 0x178, { MM_DIRECT, 1280, 800,  16, 8, 16, SEG_GRAPH } },
@@ -91,2 +94,8 @@ static struct bochsvga_mode
     { 0x18c, { MM_DIRECT, 2560, 1600, 32, 8, 16, SEG_GRAPH } },
+    { 0x18d, { MM_DIRECT, 1280, 720,  16, 8, 16, SEG_GRAPH } },
+    { 0x18e, { MM_DIRECT, 1280, 720,  24, 8, 16, SEG_GRAPH } },
+    { 0x18f, { MM_DIRECT, 1280, 720,  32, 8, 16, SEG_GRAPH } },
+    { 0x190, { MM_DIRECT, 1920, 1080, 16, 8, 16, SEG_GRAPH } },
+    { 0x191, { MM_DIRECT, 1920, 1080, 24, 8, 16, SEG_GRAPH } },
+    { 0x192, { MM_DIRECT, 1920, 1080, 32, 8, 16, SEG_GRAPH } },
 };
